website/package.json

```diff
@@ -7,7 +7,7 @@
     "build": "next build",
     "start": "next start",
     "lint": "next lint",
-    "postinstall": "yarn --cwd ../ install && yarn --cwd ../ build"
```

Comment:
I believe running the full build after install is causing an issue in CircleCI

Reply:
I'll try adding it as part of the workspace and we'll see how it works.

Reply:
I had trouble adding the website to the workspace. Though I added this command back in, and everything is working as it did previously. I think the initial issue was caused by not filtering the build process to just the `packages` folder, with `turbo build --filter=./packages/*`, so it was running build in the website in CIi. Though that shouldn't happen now with the proper filtering.